desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -1,17 +1,21 @@
 // SPDX-License-Identifier: GPL-2.0
 #include "TabDiveInformation.h"
 #include "ui_TabDiveInformation.h"
+#include "desktop-widgets/mainwindow.h" // TODO: Only used temporarilly for edit mode changes
```

Comment:
What do this TODO mean? temporarily until what?

Reply:
This will be removed once everything is undo-ized and we can remove the edit-mode. I will take care of that.